doc/EnvironmentVariables.md

```diff
@@ -0,0 +1,113 @@
+## EdgeAgent
+| Variable                             | Description                                                                                                              |
+|--------------------------------------|--------------------------------------------------------------------------------------------------------------------------|
+| BackupConfigFilePath                 | Path to put the backup deployment config file                                                                            |
```

Comment:
This is great - but can we split them into 2 categories - actual knobs that you can set, and knobs that customers should leave alone in most cases?